android/build.gradle

```diff
@@ -95,9 +95,17 @@ dependencies {
   implementation 'net.jcip:jcip-annotations:1.0'
 
   // Test Dependencies
+  testImplementation "junit:junit:$jUnitVersion"
+  testImplementation "org.mockito:mockito-core:$mockitoVersion"
+  testImplementation "org.mockito:mockito-inline:$mockitoVersion"
+
```

Comment:
Why did you return back the removed code? I assume you improperly merged it. Which tool do you use for the merge? I recommend P4Merge.